engine/src/main/java/io/zeebe/engine/state/ZeebeState.java

```diff
@@ -163,4 +166,26 @@ public MutableElementInstanceState getElementInstanceState() {
   public MutableEventScopeInstanceState getEventScopeInstanceState() {
     return eventScopeInstanceState;
   }
+
+  /**
+   * This method iterates over all entries for a given column family and presents each entry to the
+   * consumer
+   *
+   * <p><strong>Hint</strong> This method should only be used in tests
+   *
+   * @param columnFamily the enum instance of the column family
+   * @param keyInstance this instance defines the type of the column family key type
+   * @param valueInstance this instance defines the type of the column family value type
+   * @param visitor the visitor that will be called for each entry
+   * @param <KeyType> the key type of the column family
+   * @param <ValueType> the value type of the column family
+   */
+  public <KeyType extends DbKey, ValueType extends DbValue> void forEach(
+      final ZbColumnFamilies columnFamily,
+      final KeyType keyInstance,
+      final ValueType valueInstance,
+      final BiConsumer<KeyType, ValueType> visitor) {
+    final var newContext = zeebeDb.createContext();
+    zeebeDb.forEach(columnFamily, newContext, keyInstance, valueInstance, visitor);
```

Comment:
Instead of adding this to the DB just use here the column family. 

cf = zeebeDb#createCF(context, key, value)
cf.forEach(visitor)